pygfx/resources/_texture.py

```diff
@@ -275,27 +285,45 @@ def format_from_memoryview(mem, size):
 
 
 class TextureView(Resource):
-    """A view on a texture.
-
-    The view defines the sampling behavior and can specify a selection/different
-    view on the texture.
+    """View into a Texture.
+
+    Similar to numpy views, a TextureView is a view into a textures data buffer
+    with a (potentially) modified sampling behavior and can specify a
+    selection/different view on the texture.
+
+    Passing no more than ``address_mode`` and ``filter`` will create a default
+    view on the texture with the given sampling parameters.
+
+    Parameters
+    ----------
+    texture : Texture
+        The texture to view.
+    address_mode : str
+        How to handle out of bounds access. Use "clamp", "mirror" or "repeat".
+        This value can also be set per-channel by providing multiple
+        comma-separated values, e.g., "clamp,clamp,repeat".
+    filter : str
+        Interpolation mode. Possible values are: "nearest" or "linear". This
+        value can also be set using three comma-separated values to control mag,
+        min and mipmap filters individually, e.g., "linear,linear,nearest".
+    format : str
+        A format string describing the texture layout. If None, use the same as
+        the underlying texture. This must be a pygfx format specifier, e.g.
+        "3xf4", but can also be a format specific to the render backend if
+        necessary (e.g. from ``wgpu.VertexFormat``).
+    view_dim : str
+        The dimensionality of the array (1, 2 or 3). If None, use the texture's
+        dimension. Or e.g. get a "2d" slice view from a 3d texture, or e.g.
+        "cube" or "2d-array".
+    aspect : str
+        Omit or pass None to use the default.
+        @almarklein: What does this do?
```

Comment:
It's a (rather low level) wgpu thing. I don't think we use it somewhere, but for now it's not in the way either...
```suggestion
    aspect : Enum
        The `wgpu.TextureAspect` for this view. Omit or pass None to use the default.
```